src/Sulu/Bundle/AdminBundle/Resources/js/components/Select/Option.js

```diff
@@ -0,0 +1,70 @@
+// @flow
+import React from 'react';
+import classnames from 'classnames';
+import Icon from '../Icon';
+import optionStyles from './option.scss';
+
+const SELECTED_ICON = 'check';
+
+export default class Option extends React.PureComponent {
+    props: {
+        selected: boolean,
+        disabled: boolean,
+        focus: boolean,
+        value: string,
+        children?: string,
+        onClick?: (s: string) => void,
+    };
+
+    static defaultProps = {
+        disabled: false,
+        selected: false,
+        focus: false,
+        value: '',
+    };
+
+    item: HTMLElement;
+    button: HTMLButtonElement;
+
+    componentDidMount() {
+        if (this.props.focus) {
```

Comment:
Isn't only one select opened at a time? I don't really see how the problem you are describing should occur. And what do you mean with the last sentence? It's either the selected or the first one, isn't it?